web/thesauruses/c/io.json

```diff
@@ -0,0 +1,97 @@
+{
+    "meta": {
+        "language": "c",
+        "language_version": "17",
+        "language_name": "C",
+        "structure": "io"
+    },
+    "concepts": {
+        "write_line": {
+            "code": "printf(“Format Specifier”, var1, var2, …., varn);",
+            "name": "Write a line to the console"
+        },
+        "write_line_with_new_line" : {
+            "code": "printf(“Format Specifier”, var1, var2, …., varn);",
+            "name": "Write a line to the console with a return character"  
+        },
+        "read_line": {
+        "code": "scanf(“Format Specifier”, &var1, &var2, …., &varn);",
+        "name": "Read a line of user input from the console"
+        },
+        "read_char": {
+            "code": "getchar()",
+            "name": "Read a single char of user input from the console"
+            },
+        "clear_console_output": {
+            "code": "system(cls)",
+            "comment": "system() is defined in stdlib.h and cls should be in double quotes",
+            "name": "Clear the console output"
```

Comment:
You can do escape quotes (like `"system(\"cls\")"`) to add quotes where they need to be.

Also if an import is needed, you can do that too. `"code"` can be a single line (so add a `\n` in it) or an array (so 
```
"code": [
   "line1",
   "line2"
]
```
so you can include the import line and the `system` line.

Reply:
added demo code